inspirehep/config.py

```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```

Comment:
I think this should be moved to a broader discussion on `inspire-schema`, but I prefer not to block this PR.